release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2473,7 +2473,7 @@ This issue affects CPU load balancing features because these features depend on
 
 * When a node reboot occurs all pods are restarted in a random order. In this scenario it is possible that `tuned` pod started after the workload pods. This means the workload pods start with partial tuning, which can affect performance or even cause the workload to fail. (link:https://issues.redhat.com/browse/OCPBUGS-26400[*OCPBUGS-26400*])
 
-* Currently, applying a performance profile at day-0 is not supported. (link:https://issues.redhat.com/browse/OCPBUGS-18640[*OCPBUGS-18640*])
+* The installation of {product-title} might fail when a performance profile is present in the extra manifests folder and targets the master or worker pools. This is caused by the internal install ordering that processes the performance profile before the default master and worker MachineConfigPools are created. It is possible to workaround this issue by also including a copy of the stock master or worker MachineConfigPools in the extra manifests folder. (link:https://issues.redhat.com/browse/OCPBUGS-27948[*OCPBUGS-27948*]) (link:https://issues.redhat.com/browse/OCPBUGS-18640[*OCPBUGS-18640*])
```

Comment:
```suggestion
* The installation of {product-title} might fail when a performance profile is present in the extra manifests folder and targets the master or worker pools. This is caused by the internal install ordering that processes the performance profile before the default master and worker MachineConfigPools are created. It is possible to workaround this issue by including a copy of the stock master or worker MachineConfigPools in the extra manifests folder. (link:https://issues.redhat.com/browse/OCPBUGS-27948[*OCPBUGS-27948*]) (link:https://issues.redhat.com/browse/OCPBUGS-18640[*OCPBUGS-18640*])
```

Reply:
Sorry I couldn't understand the usage of "also" here. 

Should "MachineConfigPools" be in the back ticks? if not, feel free to ignore the comment!

Reply:
probably does not need also